internal/apis/certmanager/types_certificate.go

```diff
@@ -30,162 +30,217 @@ import (
 // The stored certificate will be renewed before it expires (as configured by `spec.renewBefore`).
```

Comment:
This is a copy of what we have in the public API, I just removed the // +optional annotations and the json tags.